casepropods/family_connect_registration/plugin.py

```diff
@@ -1,13 +1,51 @@
+from confmodel import fields
+from casepro.cases.models import Case
 from casepro.pods import Pod, PodConfig, PodPlugin
+import requests
 
 
 class RegistrationPodConfig(PodConfig):
-    pass
+    url = fields.ConfigText("URL to query for the registration data",
+                            required=True)
+    token = fields.ConfigText("Authentication token for registration endpoint",
+                              required=True)
+    field_mapping = fields.ConfigList(
+        "Mapping of field names to what should be displayed for them",
```

Comment:
Maybe we should specify/give an example of how this should look? Or maybe we should even have a custom config component that ensures that each element is a dict with a `field` and a `field_name` keys (although that might be overkill)